dockerfiles/base/build.sh

```diff
@@ -6,7 +6,10 @@
 # http://www.eclipse.org/legal/epl-v10.html
 
 IMAGE_NAME="eclipse/che-base"
-. $(cd "$(dirname "$0")"; pwd)/../build.include
+base_dir=$(cd "$(dirname "$0")"; pwd)
+. $base_dir/../build.include
 
 init "$@"
 build
+
+source $base_dir/all_tests.sh
```

Comment:
Ok, understand. I did not realize there could be users who would develop/run from source eclipse-che and would not have bash, only Bourne shell.